app/src/main/res/layout/activity_chat.xml

```diff
@@ -54,7 +65,7 @@
                     android:padding="3dp"
                     android:fontFamily="@font/armata"
                     android:hint="Type your post"
-                    android:inputType="textPersonName"
+                    android:inputType="textMultiLine"
```

Comment:
What's the aim of changing this input type ?

Reply:
The purpose is here to have a message appearing on several lines and not on a single line only. This is visually more appealing and of course much cleaner. I don't know what is the specificity of "textPersonName" (it was initially written by Roxane), but that's definitely not the inputType to use here.

Reply:
Ok, great, I understand better thanks!